unified_planning/model/action.py

```diff
@@ -90,6 +91,13 @@ def name(self, new_name: str):
         """Sets the parameter name."""
         self._name = new_name
 
+    def cost(self) -> Optional['up.model.fnode.FNode']:
```

Comment:
This is OK. Another possible design would be to have a map inside the plan quality metric from actions to costs. Maybe we can discuss this in next WP3 meeting.